boards/arm/ti_am62x_sk_m4/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _ti_am62x_sk_m4:
+
+AM62x-SK M4F Core
+#################
+
+Overview
+********
+
+The AM62x-SK board configuration is used by Zephyr applications that run on
+the TI AM62x platform. The board configuration provides support for the ARM
+Cortex-M4F CPU and the following devices:
```

Comment:
Better to say "Cortex-M4F microcontroller/MCU core" than Cortex-M4F "CPU"?

Reply:
Made the change. Thanks for the suggestion!